Tests/DisabledTests/BSONValueTests.swift

```diff
@@ -1,11 +1,9 @@
-import CLibMongoC
+@testable import BSON
```

Comment:
given that this file is still in "disabled", does it make sense to include these changes now or no?